### ISSUE TITLE
Nasal / Medial location

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11562]
Nasal / Medial location (Complement to Temporal)

Preferred term label:
Nasal / medial location

Synonyms

    Nasal location

    Medial location

    Nasally located

Definition (free text, please give PubMed ID)
An anatomical spatial pattern or localization referring to the medial side of the body, head, or eye, positioned towards the midline or the nose. In ophthalmology, this modifier is frequently used to describe the spatial distribution of manifestations such as nasal pterygia, nasal visual field defects (nasal step), or nasal retinal thinning. PMID:29088514, PMID:25026462

Parent term (use hpo.jax.org/app)
The anatomical localization of the specified phenotypic abnormality (HP:0012830)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)

    Glaucoma (OMIM:137760) (e.g., nasal step visual field defect)

    Pterygium (OMIM:178000)

Your nano-attribution (ORCID)
0000-0001-9042-0508
